### DISCUSSION TITLE
Welcome to zicBox Discussions!

### DISCUSSION THREAD

**Discussion by apiel** [apiel/zicBox#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Hey guys.  I'm Volard from Canada.  I fell in love with this project since it was on Floyd Steinberg's youtube.  It was much different then and much more improved now.  I am a hobbiest DIY person.  I have some basic electronics skills, very little coding but a fair amount of 3D printing and newly 3D design.  
Currently I'm working on a case loosely inspired by the Gameboy Advance (the wide one, not the SP that closes the screen vertically).  My idea was to make it an all in one groovebox that I can use on the couch or in bed whenever the mood struck me.  I'm adding 2 3.7v lithium rechargeable cells (salvaged from vapes), stereo laptop speakers in thier own enclosure, micro amplifier with volume control (based on the PAM8403 amp chip, USB charging circuit, extended headphone jack for private beatmaking.  I'm still early in the prototype design phase but the unit is comfy and as small as I could make it and still fit all those goodies in.
I'm posting over on the discord with updates.  If you're not already a member, come check it out.  Some great ppl on there.
